app/assets/stylesheets/common/base/edit-topic-timer-modal.scss

```diff
@@ -7,9 +7,18 @@
     border-top: 0;
     padding: 10px 0;
   }
+  .modal-inner-container {
+    box-sizing: border-box;
+    min-width: 310px;
+  }
   .modal-body {
     max-height: none;
     width: 375px;
+    box-sizing: border-box;
+
+    @media (max-width: 600px) {
+      width: 100% !important;
```

Comment:
I'm not really good with CSS but I've gotten reviews in the past where we don't encourage the use of `!important`

Reply:
Yes I was being cheeky...I actually took a second look and found why this wasn't working without important, pushing that change now

Reply:
> Yes I was being cheeky...I actually took a second look and found why this wasn't working without important, pushing that change now

shame shame shame